src/globals/zothacks-helper.scss

```diff
@@ -0,0 +1,5 @@
+@mixin zothacks-background {
+    background: url(../../../assets/images/background_terrain.webp);
```

Comment:
WebP seems to work great, but it is unsupported on some older versions of Safari, so we'll need to add a fallback, perhaps with the [image-set](https://developer.mozilla.org/en-US/docs/Web/CSS/image/image-set) function.

Reply:
I think it'll work with just the 1.25 MB jpg (no WebP at all). I'm having difficulty finding a balance for image-set between various browsers - for one, Chrome doesn't allow image-set selecting a file based on type, only resolution :(